dev_docs/cxg.md

```diff
@@ -1,5 +1,3 @@
-## UPDATE (9/30/2020): Starting today, the name Corpora will only be used as the internal project name, with cellxgene Data Portal being the official product name
-
```

Comment:
Seemed old enough to now remove

Reply:
Updated date, but I'm not going to update the version: See below [comment](https://github.com/chanzuckerberg/single-cell-explorer/pull/49/files#r707791218) and further explanation that I've since added in `CXG Version History` section.